core/src/replay_stage.rs

```diff
@@ -8147,7 +8155,10 @@ pub(crate) mod tests {
         assert_eq!(reset_fork, Some(6));
         assert_eq!(
             failures,
-            vec![HeaviestForkFailures::FailedSwitchThreshold(4, 0, 30000),]
```

Comment:
previously we would have passed the lockout check, bc we check `6` instead of `4`.